### ISSUE TITLE
tune hyperparameters using ngram & mlp model

### ISSUE THREAD

**Issue by godekdls** [godekdls/korean-thematic-analysis#29]
| samples per class 	| learning rate 	| epoch 	| batch size 	| layers 	| units 	| dropout rate 	|  features 	| accuracy 	|    variance   	|
|:-----------------:	|:-------------:	|:-----:	|:----------:	|:------:	|:-----:	|:------------:	|:---------:	|:--------:	|:-------------:	|
|        300        	|      1e-3     	|  1000 	|     128    	|    2   	|   64  	|      0.2     	|   20000   	|  63.92%  	|  over fitting 	|
|        1000       	|      1e-3     	|  1000 	|     128    	|    2   	|   64  	|      0.2     	|   20000   	|  71.98%  	|  over fitting 	|
|        1000       	|      1e-3     	|  1000 	|     128    	|    2   	|   64  	|    **0.5**   	|   20000   	|  72.45%  	|  over fitting 	|
|        1000       	|      1e-3     	|  1000 	|     128    	|    2   	|   64  	|      0.5     	| **15000** 	|  72.10%  	|  over fitting 	|
|        1000       	|      1e-3     	|  1000 	|     128    	|  **1** 	|   64  	|      0.5     	|    2000   	| too slow 	| under fitting 	|
|        1000       	|    **1e-2**   	|  1000 	|     128    	|    1   	|   64  	|      0.5     	|    2000   	| too slow 	| under fitting 	|
|        1000       	|    **1e-1**   	|  1000 	|     128    	|    1   	|   64  	|      0.5     	|    2000   	|  69.72%  	|  over fitting 	|
